### ISSUE TITLE
Staging Review finding: Links don't indicate their current state

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#118759]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** vfs-mhv-secure-messaging
**Product:** mhv-secure-messaging
**Feature:** Oracle Health curated list

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

After sending a message, the group of links (inbox, sent, more folders) have the visual treatment of tabs, and the current selected link has a visual affordance, but the state ("current") is not indicated programmatically to assistive technologies. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="741" height="333" alt="pseudo-tabs" src="https://github.com/user-attachments/assets/30e14854-3b7d-4f8a-867c-8ef123b048d8" />

</details>

## Recommended action

Either use the tabs component, or ensure that the links indicate state using `aria-current="true"`. 
Note: tabs use buttons, as buttons are for in-page actions. It appears that these links are navigating to a new page, hence the recommendation. 

[Tabs component](https://design.va.gov/components/tabs)

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG22/#name-role-value)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
